domain-models-runtime/src/main/java/org/folio/rest/tools/client/HTTPJsonResponseHandler.java

```diff
@@ -64,21 +62,6 @@ public void handle(AsyncResult<HttpResponse<Buffer>> res) {
         log.error(e.getMessage(), e);
       }
     }
-      /* if(r.error != null && rollbackURL != null){
-
-      }*/
-
-    if (webClient != null) {
```

Comment:
I don't see any place where `webclient` class field have been initialized or used. I beleive it could be removed from class definition as well

Reply:
It's set in HttpModuleClient2 .. when autoCloseConnections = true.